rpm/containerd.spec

```diff
@@ -30,8 +30,8 @@ AutoReq: no
 
 Name: containerd.io
 Provides: containerd
-# For some reason on rhel 8 if we "provide" runc then it makes this package unsearchable
-%if 0%{!?el8:1}
+# For some reason on rhel >= 8 if we "provide" runc then it makes this package unsearchable
+%if %{undefined rhel} || 0%{?rhel} >= 8
```

Comment:
Went for a slightly more verbose / less hard to graph approach, using `undefined`